test/test_regression_cli.py

```diff
@@ -109,7 +116,11 @@ def test_csv_regression(dataset, expected, tmpdir):
 
 @pytest.mark.parametrize("dataset, expected", download_md5)
 def test_download_regression(dataset, expected):
-    """Check for regression for a particular dataset downloaded only"""
+    """Check for regression for a particular dataset downloaded only
+
+    move the dataset script downloaded to the .retriever file before download
```

Comment:
Good question. So the scripts search is current directory, and then scripts directory and .retriever directory. 

In the case of the other tests using `imp.load_module` we give them the path to the script directory. In download only tests, since the scripts cant be  hit 
there are two ways 
Either cd .. to move from current directory` test` to retriever main directory or to copy the scripts in to the .retriever directory.